doc/nrf/releases_and_maturity/releases/release-notes-2.6.0.rst

```diff
@@ -0,0 +1,1525 @@
+   * New device types:
+
+     * Refrigerator
+     * Room air conditioner
+     * Dishwasher
+     * Laundry washer
+     * Robotic vacuum cleaner
+     * Smoke CO alarm
+     * Air quality sensor
+     * Air purifier
+     * Fan
+
+   * Product Appearance attribute in the Basic Information cluster that allows describing the product's color and finish.
+
+* Updated:
+
+  * The following Kconfig options have been renamed:
+
+   * ``CONFIG_CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT`` to :kconfig:option:`CONFIG_CHIP_ENABLE_ICD_SUPPORT`.
+   * ``CONFIG_CHIP_SED_IDLE_INTERVAL`` to :kconfig:option:`CONFIG_CHIP_ICD_SLOW_POLL_INTERVAL`.
+   * ``CONFIG_CHIP_SED_ACTIVE_INTERVAL`` to :kconfig:option:`CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL`.
+   * ``CONFIG_CHIP_SED_ACTIVE_THRESHOLD`` to :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_THRESHOLD`.
+
+   * zcbor 0.7.0 to 0.8.1.
+
+   * The :kconfig:option:`CONFIG_PRINTK_SYNC` Kconfig option has been disabled to avoid potential interrupts blockage in Matter applications that can violate time-sensitive components, like :ref:`nrf_802154`.
+
+* The Factory Data Guide regarding the changes for the SPAKE2+ verifier, and generation of the Unique ID for Rotating Device ID.
+* The factory data default usage has been changed:
+
+  * The SPAKE2+ verifier is now generated by default with each build, and it will change if any of the related Kconfig options are modified.
+    This resolves the :ref:`known issue <known_issues>` related to the SPAKE2+ verifier not regenerating (KRKNWK-18315).
+  * The Test Certification Declaration can now be generated independently from the generation of the DAC and PAI certificates.
+  * The Unique ID for Rotating Device ID can now only be generated if the :kconfig:option:`CONFIG_CHIP_ROTATING_DEVICE_ID` Kconfig option is set to ``y``.
+
+Thread
+------
+
+* Updated:
+
+  * The default cryptography backend for Thread is now Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
+    You can still build all examples with deprecated Mbed TLS support by setting the :kconfig:option:`CONFIG_OPENTHREAD_NRF_SECURITY_CHOICE` Kconfig option to ``y``, but you must build the Thread libraries from sources.
+    To :ref:`inherit Thread certification <ug_thread_cert_inheritance_without_modifications>` from Nordic Semiconductor, you must use the PSA Crypto API backend.
+
+  * nRF5340 SoC targets that do not include :ref:`Trusted Firmware-M <ug_tfm>` now use Hardware Unique Key (HUK, see the :ref:`lib_hw_unique_key` library) for PSA Internal Trusted Storage (ITS).
+
+See `Thread samples`_ for the list of changes in the Thread samples.
+
+Zigbee
+------
+
+* Updated:
+
+  * :ref:`nrfxlib:zboss` to v3.11.3.0 and platform v5.1.4 (``v3.11.3.0+5.1.4``).
+    They contain fixes for security vulnerabilities and other bugs.
+    For details, see :ref:`zboss_changelog`.
+  * :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.1.2.
```

Comment:
```suggestion
  * :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.2.2.
```

Please fix this typo. Correct NCP host version is here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss/